lib/engines/content_block_manager/app/views/shared/_phase_banner.html.erb

```diff
@@ -1,13 +1,13 @@
 <%= render "govuk_publishing_components/components/phase_banner", {
-    phase: "Alpha",
-    message: sanitize("This is a new service - your
+    phase: "Beta",
```

Comment:
There's a test here that'll need changing once this is rebased https://github.com/alphagov/whitehall/blob/484509f1f3f11d2725447fee5b0a4aff6671e97c/test/functional/admin/errors_controller_test.rb#L59